### DISCUSSION TITLE
Upstream in Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by cod3mas0n** [NginxProxyManager/nginx-proxy-manager#3558] [Q&A]
Is there any way to add and use the upstream?
I've set upstream in `custom/http_top.conf` but when I'm using the upstream name in UI, the proxy host goes offline, think missed something! or do I need to map it to the $server?  

```
upstream backend {
  server 192.168.0.111;
  server 192.168.0.112;
  server 192.168.0.113;
}
```

**Comment by likelios** [comment]
Yes there is some description here
but how exactly to do this no!

[/data/nginx/custom/stream.conf](https://nginxproxymanager.com/faq/#i-can-t-get-my-service-to-proxy-properly)

**Therefore, I ask the respected developer to look towards resolving this issue.**
